quick-start/examples/n5.min.js

```diff
@@ -0,0 +1,21 @@
+import { Scene, Circle, Rectangle } from "https://unpkg.com/pencil.js/dist/pencil.esm.js";
+
+const scene = new Scene(undefined, {
+    fill: "#45a8ff",
+});
+
+const sun = new Circle(scene.center, 100, {
+    fill: "orange",
+});
+
+const width = scene.width;
+const height = scene.height / 2;
+// Positions can be declared with a Position object or an array
+const ocean = new Rectangle([0, scene.height / 2], width, height, {
+    fill: "#28488f",
+});
+
+// The order determine the how shape will overlap
```

Comment:
```suggestion
// Later components will overlap former components
```
The text doesn't explain how the order affects the overlap of components.
Also consider moving this comment directly above the line with `.add()` with the indentation.